frontend/src/metabase/auth/components/PasswordButton/PasswordButton.tsx

```diff
@@ -2,16 +2,26 @@ import React from "react";
 import { t } from "ttag";
 import AuthButton from "../AuthButton";
 
-interface PasswordButtonProps {
+export interface PasswordButtonProps {
+  isLdapEnabled: boolean;
   redirectUrl?: string;
 }
 
-const PasswordButton = ({ redirectUrl }: PasswordButtonProps) => {
+const PasswordButton = ({
+  isLdapEnabled,
+  redirectUrl,
+}: PasswordButtonProps) => {
   const link = redirectUrl
     ? `/auth/login/password?redirect=${encodeURIComponent(redirectUrl)}`
     : `/auth/login/password`;
 
-  return <AuthButton link={link}>{t`Sign in with email`}</AuthButton>;
+  return (
+    <AuthButton link={link}>
+      {isLdapEnabled
+        ? t`Sign in with username or email`
```

Comment:
Not sure about the copy update if it's been discussed somewhere.

Reply:
This is to mirror the following logic https://github.com/metabase/metabase/blob/b50b06ecce3c1a6992a7194a8c0da51c948f7191/frontend/src/metabase/entities/users/forms.js#L141